cleanlab_studio/cli/dataset/schema.py

```diff
@@ -1,4 +1,3 @@
-import pathlib
```

Comment:
I'd like to deprecate this module altogether. We are moving schema validation to the backend, so users shouldn't be generating/validating schemas at all